openedx/core/lib/xblock_builtin/xblock_discussion/static/js/mathjax_include.js

```diff
@@ -0,0 +1,35 @@
+var vendorScript;
+if (typeof MathJax === 'undefined') {
+    vendorScript = document.createElement('script');
+    vendorScript.onload = function() {
+        'use strict';
+        var MathJax, setMathJaxDisplayDivSettings;
```

Comment:
This `var MathJax` declaration is masking the `window.MathJax` that was defined by the loaded script, and so `MathJax` ends up `undefined`.  Please change this line to:

`var setMathJaxDisplayDivSettings;`

Or alternately, set `var MathJax = window.MathJax`;